pkgs/by-name/al/algia/package.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "algia";
+  version = "0.0.74";
+
+  src = fetchFromGitHub {
+    owner = "mattn";
+    repo = "algia";
+    rev = "v${version}";
+    sha256 = "sha256-t6XDw40FTa7QkZmOkgAufWV1aFjQrLWmycp+zcVYQWs=";
+  };
+
+  vendorHash = "sha256-fko9WC/Rh5fmoypqBuFKiuIuIJYMbKV+1uQKf5tFil0=";
+
+  meta = with lib; {
```

Comment:
Better to avoid `with lib`.

Reply:
Then should I write such as below?
```
  meta = {
    description = "A cli application for nostr";
    homepage = "https://github.com/mattn/algia";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ haruki7049 ];
  };
```

Reply:
resolved: 49b84d920332672e2eab9df0fbfcdf82ec67c3df